### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90

### ISSUE THREAD

**Issue by rogeriobastos** [CycloneDX/cyclonedx-python#611]
Hi,

cyclonedx-python fails with the following error while analyse `poetry.lock` file because there are a special character in description filed of `pyhumps` package. 

```
Traceback (most recent call last):
  File "/usr/bin/cyclonedx-py", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.11/site-packages/cyclonedx_py/client.py", line 343, in main
    CycloneDxCmd(args).execute()
  File "/usr/lib/python3.11/site-packages/cyclonedx_py/client.py", line 148, in execute
    output = self.get_output()
             ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/cyclonedx_py/client.py", line 85, in get_output
    parser = self._get_input_parser()
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/cyclonedx_py/client.py", line 296, in _get_input_parser
    input_data = input_data.decode(input_encoding)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/encodings/cp1252.py", line 15, in decode
    return codecs.charmap_decode(input,errors,decoding_table)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 116633: character maps to <undefined>
```

poetry.lock snippet

```
[[package]]
name = "pyhumps"
version = "3.7.1"
description = "🐫  Convert strings (and dictionary keys) between snake case, camel case and pascal case in Python. Inspired by Humps for Node"
optional = false
python-versions = "*"
files = [
    {file = "pyhumps-3.7.1-py3-none-any.whl", hash = "sha256:c6f2d833f2c7afae039d71b7dc0aba5412ae5b8c8c33d4a208c1d412de17229e"},
    {file = "pyhumps-3.7.1.tar.gz", hash = "sha256:5616f0afdbc73ef479fa9999f4abdcb336a0232707ff1a0b86e29fc9339e18da"},
]
```

**Comment by jkowalleck** [comment]
@rogeriobastos please upload the poetry lock file, for reproducibility, and so that I can tailor a regression test from it.

**Comment by rogeriobastos** [comment]
[poetry.zip](https://github.com/CycloneDX/cyclonedx-python/files/13241451/poetry.zip)

**Comment by jkowalleck** [comment]
the issue should be fixed with v3.11.6
